test/Interop/Cxx/namespace/Inputs/classes.h

```diff
@@ -40,4 +40,21 @@ struct BasicStruct {
 };
 } // namespace ClassesNS3
 
+namespace GlobalAliasToNS1 = ClassesNS1;
+
+namespace ClassesNS4 {
+namespace AliasToGlobalNS1 = ::ClassesNS1;
+namespace AliasToGlobalNS2 = ::ClassesNS1::ClassesNS2;
+
+namespace ClassesNS5 {
+struct BasicStruct {};
+} // namespace ClassesNS5
+
+namespace AliasToInnerNS5 = ClassesNS5;
+namespace AliasToNS2 = ClassesNS1::ClassesNS2;
+
+namespace AliasChainToNS1 = GlobalAliasToNS1;
+namespace AliasChainToNS2 = AliasChainToNS1::ClassesNS2;
+} // namespace ClassesNS4
+
```

Comment:
Are more complex test cases covered by other tests? For example, consider code with namespaces that are shadowed

```
namespace X { }
namespace Y { namespace X { }; namespace Z = X; /* refers to ::Y::X */ }
```

or code with `using namespace ABC`. Do these get translated correctly?

Reply:
There wasn't a test for such case, but I've added it now. Also added a test for an alias pointing to its parent namespace, and a couple of other similar cases.

Using-directives & using-declarations currently aren't imported at all (these C++ constructs, along with inline namespaces, are among those that I don't know how to express ergonomically in Swift).